spec/rubocop/cop/rspec/instance_variable_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe RuboCop::Cop::RSpec::InstanceVariable do
+RSpec.describe RuboCop::Cop::RSpec::InstanceVariable, :config do
```

Comment:
Because it needs the default config, specifically `Include`, otherwise
```ruby
expect_no_offenses(<<-RUBY, 'lib/source_code.rb')
```
fails.